pallets/registrar/src/lib.rs

```diff
@@ -731,8 +744,7 @@ pub mod pallet {
                 ) -> (T::AccountId, DepositBalanceOf<T>) {
                     const SEED: u32 = 0;
                     let user = account(string, n, SEED);
-                    T::Currency::make_free_balance_be(&user, total);
-                    let _ = T::Currency::issue(total);
+                    assert_ok!(T::Currency::mint_into(&user, total));
```

Comment:
Is this effectively the same?

Reply:
Since we're creating a new user it's previous balance is zero, so both seems equivalent.
However calling 2 time `create_funded_user` with the same args will have a different behavior (the old one actually increasing the supply while the balance don't increase).